pinot-spi/src/main/java/org/apache/pinot/spi/stream/PartitionGroupMetadataFetcher.java

```diff
@@ -63,7 +63,8 @@ public Exception getException() {
   @Override
   public Boolean call()
       throws Exception {
-    String clientId = PartitionGroupMetadataFetcher.class.getSimpleName() + "-" + _topicName;
+    String clientId = PartitionGroupMetadataFetcher.class.getSimpleName() + "-"
```

Comment:
I wonder if we should also add a timestamp here since this is part of a periodic task. Shouldn't matter much as the usage here is limited to metadata lookup.

Reply:
Wouldn't table name itself give that uniqueness ?